db/migrate/20141015173220_add_docker_image_fields.rb

```diff
@@ -1,35 +0,0 @@
-class AddDockerImageFields < ActiveRecord::Migration[4.2]
-  def up
-    add_column :docker_images, :katello_uuid, :string, :limit => 255
-    add_column :docker_images, :katello_repository_id, :integer
-    add_column :docker_tags, :katello_repository_id, :integer
-
-    create_table :katello_repository_docker_images do |t|
-      t.references :docker_image, :null => false
-      t.references :repository, :null => true
-    end
```

Comment:
@bastilian hey, sorry for the delay.  The history of the katello_repository_docker_images is this:
2014 - created with reference to to the katello_docker_image  table
2016 - Dropped (during our docker v1 to v2 migration)

This table didn't refer to anything in foreman_docker so i don't think we want to drop this table, nor modify any reference to.

Reply:
errr, actually @parthaa, the foriegn key below this seems to refer to the foreman_docker table 'docker_images':

```
    add_foreign_key :katello_repository_docker_images, :docker_images,
                    :column => :docker_image_id
```

which seems contrary to what we discussed last week.